app/views/hosts/_operating_system.html.erb

```diff
@@ -9,26 +9,18 @@
 
 <div id='provisioning_method' <%= display? (@host.capabilities.size < 2)  %> >
   <%= field(f, :provision_method, :label => _('Provisioning Method'), :required => true) do
-      radio_button_f(f, :provision_method, :value=>'build', :checked=> @host.pxe_build?, :text=> _("Network Based"), :disabled => !@host.new_record?)+
-      radio_button_f(f, :provision_method, :value=>'image', :checked=> @host.image_build?, :text=> _("Image Based"), :disabled => !@host.new_record?)
+    @host.provision_methods.collect do |provision_method, friendly_name|
```

Comment:
Consider sorting this either here in the view or in the model, else it might move around depending on how plugins register.